### ISSUE TITLE
Add historical commits

### ISSUE THREAD

**Issue by tobiasvl** [EngineerSmith/nativefs#2]
Hey! This repo seems to be the most popular fork of the original.

I noticed two things though:

1. Your repo doesn't have the actual commit history of the original history (although the initial commit mentions the SHA of the original master), unlike https://github.com/zorggn/nativefs
2. Your repo is ahead of https://github.com/zorggn/nativefs

Is there any possibility of rewriting your history to include the original commits, or are they lost?

**Comment by EngineerSmith** [comment]
Sorry for the late reply - I was taking a week off. 

I've looked through my files and this repo was originally a submodule - which means I don't have the original .git for it as I moved it to use this repo now (not sure if it's hidden within the parent's .git somewhere). However I did manage to find another version with additional history, c01aea7, which is a ahead of Zorggn's by 8 commits so there is some additional history that could be savaged (was latest on the 22nd Feb 2021). 

I'm not a git whizz, so I'm not sure how to transfer this history over; hence the basic commit originally. I've attached it so it can still be accessed
[nativefs-c01aea7.zip](https://github.com/EngineerSmith/nativefs/files/8990593/nativefs-c01aea7.zip)
